packages/frontend/src/features/sqlRunner/store/sqlRunnerSlice.ts

```diff
@@ -117,6 +124,15 @@ export const sqlRunnerSlice = createSlice({
         },
         setActiveEditorTab: (state, action: PayloadAction<EditorTabs>) => {
             state.activeEditorTab = action.payload;
+            if (action.payload === EditorTabs.VISUALIZATION) {
+                state.activeSidebarTab = SidebarTabs.VISUALIZATION;
+                if (state.selectedChartType === undefined) {
```

Comment:
nice we could even do this depending on the data available at some point

Reply:
absolutely!